modules/node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id="node-tuning-hosted-cluster_{context}"]
+= Configuring node tuning in a hosted cluster (Technology Preview)
+
+//# Manage node-level tuning with the Node Tuning Operator
+
+To set node-level tuning on the nodes in your hosted cluster, you can use the Node Tuning Operator. In hosted control planes, you can configure node tuning by creating config maps that contain tuned objects and referencing those config maps in your node pools. Currently, node tuning is limited to actions that the TuneD daemon can apply directly, such as setting sysctl values. Tuning that requires setting kernel boot parameters is not yet supported for hosted control planes.
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+. Create a config map that contains a valid tuned manifest, and reference the manifest in a node pool. In the following example, a tuned manifest defines a profile that sets `vm.dirty_ratio` to 55 on nodes that contain the `tuned-1-node-label` node label with any value. Save the config map manifest in a file named `tuned-1.yaml`:
+
```

Comment:
A couple things here. 
* Use a `+` sign to nest the code block beneath the step, as you have with the `[NOTE]` block. 
* For coloration on docs.openshift, include a [source,yaml] block. 
* You also need four dashes around the code block, before and after. 

Example: 
```
Save the config map manifest in a fila named `tuned-1.yaml`:
+
[source,yaml]
----
apiVersion:
etc...
----
+
[NOTE]
```

Reply:
So this is MUCH better, the build preview looks great! I noticed you still have empty lines between the step and the output on lines 18, 47, 62, 96, 113, and 122. While this still renders correctly on the docs.openshift build preview, there is no guarantee that it will render accurately on the customer portal or other places we publish.  I recommend removing empty lines where possible.

The second header and procedure needs to be in it's own module. Please trim the empty lines that are currently on lines  179, 194, 211, 229, 253, 270, and 290.